app/src/main/admin/actions/usersActionCreators.ts

```diff
@@ -16,4 +16,30 @@ export const usersActionCreators = {
             } as AllUsersFetched);
         }
     },
+
+    createUser(name: string, email: string, username: string) {
+        return async (dispatch: Dispatch<AdminAppState>, getState: () => AdminAppState) => {
+            const result = await (new UsersService(dispatch, getState)).createUser(name, email, username);
+            if (result && result.errors) {
+                dispatch({
+                    type: UsersTypes.SET_CREATE_USER_ERROR,
+                    error: result.errors[0].message
```

Comment:
Would be better to handle all errors coming back from the API, not just the first one.